Providers/Scripts/3.x/Scripts/nxFile.py

```diff
@@ -83,6 +83,12 @@ def Test_Marshall(DestinationPath, SourcePath, Ensure, Type, Force, Contents, Ch
 
 
 def Get_Marshall(DestinationPath, SourcePath, Ensure, Type, Force, Contents, Checksum, Recurse, Links, Owner, Group, Mode):
+
+    f = open("/home/zia/test_folder/python.txt", "a")
+    f.write("DestinationPath = " + DestinationPath)
+    f.write("Ensure = " + Ensure)
```

Comment:
i think its unused code. please delete it

Reply:
Thanks for catching it!

---
In reply to: [263605536](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/541#discussion_r263605536) [](ancestors = 263605536)